AppCreationScripts/apps.json

```diff
@@ -27,7 +27,34 @@
             }
           ]
         }
-      ]
-    }
+      ],
+      "codeConfigurations": [	
+          {
+            "settingFile": "/app/src/main/res/raw/auth_config_multiple_account.json",
+            "replaceTokens": {
+                "appId": "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
```

Comment:
Should this be replaced with the users information? "Enter_the_application_Id_here"? Ditto for redirect uri, package name, and key hash

Reply:
I'm guessing this might means "replace these key pair values", but I'll let @archieag to confirm.

Reply:
This config informs portal what values to replace. For ex, it'll know that appId needs to be updated in this settings file. And, then, it'll look the value of this appId and replace with one in the portal.
So, it needs to match what's in the settings file.